whichone/static/js/data.js

```diff
@@ -83,3 +83,13 @@ async function getSpotifyData() {
   await request("top_artists", compareArtists);
   await request("top_tracks", getAudioFeatures);
 }
+
+
+function deleteAllCookies() {
+  const cookies = document.cookie.split(";");
+  cookies.forEach(cookie => {
+    const eqPos = cookie.indexOf("=");
+    const name = eqPos > -1 ? cookie.substr(0, eqPos) : cookie;
+    document.cookie = name + "=;expires=Thu, 01 Jan 1970 00:00:00 GMT";
+  });
+}
```

Comment:
Have you tried clearing the cookies using this API instead of the old-fashioned way?
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/browsingData/removeCookies

Reply:
I've done some further digging and it seems like we don't actually need this function at all because the session cookies are reset by the flask app in python